trace-dispatcher/src/Control/Tracer/Arrow.hs

```diff
@@ -0,0 +1,90 @@
+{-|
+Module      : Control.TracerA.Arrow
```

Comment:
I wrote this in the README:
"We integrated contra-tracer-0.1.0.0 into the source tree, because the
iohk-monitoring framework depends on the non-arrow based contra-tracer framework.
This should become a dependency later".
So after old tracing will be removed it will become a dependeny.

Reply:
That's ok then.